LiteCore/RevTrees/RawRevTree.cc

```diff
@@ -29,6 +29,29 @@ namespace litecore {
 #pragma pack()
 
 
+    bool RawRevision::isRevTree(slice raw_tree) {
+        // The data cannot be shorter than a single revision:
+        if (raw_tree.size < sizeof(RawRevision))
+            return false;
+        // Data must end with a 32-bit 0:
+        auto end = (const void*)offsetby(raw_tree.end(), -sizeof(uint32_t));
```

Comment:
"The result of a unary negative operator on an unsigned type is still unsigned" according to Windows.